spec.html

```diff
@@ -125,6 +125,61 @@ <h1>Updated Productions</h1>
   </emu-clause>
 </emu-clause>
 
+<emu-clause id="sec-new-ecmascript-specification-types">
+  <h1>New ECMAScript Specification Types</h1>
+
+  <emu-clause id="sec-classes-specification-types">
+    <h1>Classes Specification Types</h1>
+
+    <emu-clause id="sec-elementdescriptor-specification-type" aoid=ElementDescriptor>
+      <h1>The ElementDescriptor Specification Type</h1>
+
+      <p>
+        The ElementDescriptor is a Record used to represent class elements at runtime.
+        Values of the ElementDescriptor type are Record values whose fields are defined as by <emu-xref href="#table-1"></emu-xref>.
+      </p>
+
+      <emu-table id="table-1" caption="ElementDescriptor fields">
```

Comment:
Table `id` values should be descriptive. You should use something like `table-element-descriptor-fields`.